src/shared/tpm2-util.c

```diff
@@ -480,6 +480,56 @@ void tpm2_pcr_mask_to_selection(uint32_t mask, uint16_t bank, TPML_PCR_SELECTION
         };
 }
 
+static void tpm2_log_debug_buffer(const void *buffer, const size_t size, const char *msg) {
+        if (!DEBUG_LOGGING || !buffer || !size)
+                return;
+
+        _cleanup_free_ char *h = hexmem(buffer, size);
+        if (h)
+                log_debug("%s: %s", msg ?: "Buffer", h);
+        else
+                log_oom();
```

Comment:
`log_oom_debug()`

Or, we usually silently ignore the failure in logging, and wrap `h` with `strna()`:
```
_cleanup_free_ char *h = hexmem(buffer, size);
log_debug("%s: %s", msg ?: "Buffer", strna(h));
```

Reply:
i changed it to use strna(), but i wonder if it would be useful to add stroom() that could be used instead, so it's clear that it's a OOM error